src/test-run/commands/index.js

```diff
@@ -188,6 +188,23 @@ export class TypeTextCommand extends Assignable {
     }
 }
 
+export class NavigateToCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type = TYPE.navigateTo;
+        this.url  = null;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'url', type: nonEmptyStringArgument, required: true }
```

Comment:
Ok, then we need tests for 3 scenarios: 
- URL with protocol 
- URL with relative protocol 
- relative URL